client/store/reports/items/selectors.js

```diff
@@ -21,7 +21,7 @@ import { getJsonString } from 'store/utils';
  * @return {Object}           Report details
  */
 function getReportItems( state, endpoint, query = {} ) {
-	return get( state, [ 'reports', 'items', endpoint, getJsonString( query ) ], [] );
+	return get( state, [ 'reports', 'items', endpoint, getJsonString( query ) ], {} );
```

Comment:
Do you think it would be worthwhile to make the default object `{ data: [] }` ?

Reply:
Sounds good! :+1: